cmd/sandbox/start.go

```diff
@@ -86,7 +110,8 @@ func startSandbox(ctx context.Context, cli docker.Docker, reader io.Reader) (*bu
 		if err.Error() != clierrors.ErrSandboxExists {
 			return nil, err
 		}
-		printExistingSandboxMessage()
+		fmt.Printf("Existing details of your sandbox:")
```

Comment:
Move this inside the function on L114

Reply:
no `util.PrintSandboxMessage()` is a common function....It also print message on new cluster